templates/deb/postinst.erb

```diff
@@ -44,3 +43,13 @@ fi
     fi
   <%- end -%>
 <%- end -%>
+
+# Run postinstall scripts on install if defined
+if [ $1 = configure ] && [ -z $2] ; then
```

Comment:
This bash needs some tweaking. The test needs a space between `$2` and `]` to be valid.